### DISCUSSION TITLE
Frontend Framework/Library

### DISCUSSION THREAD

**Discussion by Malix-Labs** [scratch-for-discord/Web-Application_Frontend#169] [Polls]
Which frontend framework/library would you prefer S4D to be coded in?

_**The simple result of this poll will not chose which framework we will use**_

**Comment by nekoify** [comment]
more rigged then the election 🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬

**Reply by mrredo** [reply]
Yes

**Comment by Dark1Mole** [comment]
**_r u serios raight niow brou_**

**Comment by Malix-Labs** [comment]
Well, fuck.
